rcl_yaml_param_parser/include/rcl_yaml_param_parser/parser.h

```diff
@@ -119,6 +119,17 @@ rcl_variant_t * rcl_yaml_node_struct_get(
   const char * param_name,
   rcl_params_t * params_st);
 
+/// \brief Get the descriptor struct for a given parameter
+/// \param[in] node_name is the name of the node to which the parameter belongs
+/// \param[in] param_name is the name of the parameter whose value is to be retrieved
+/// \param[inout] params_st points to the populated (or to be populated) parameter struct
+/// \return parameter descriptor struct on success and NULL on failure
```

Comment:
@bpwilcox what if no descriptor was specified? what will this function do?

Reply:
If the parameter name or node name does not exist for the descriptor, then it will still return a descriptor (not NULL). I found that this is the same behavior as what happens with `rcl_yaml_node_struct_get`.